src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 3;
+  const longTerm = 7;
+
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  const rentalPerDay = 40;
+  const rentalPrice = rentalPerDay * days;
+
+  if (days >= longTerm) {
+    return rentalPrice - longTermDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return rentalPrice - shortTermDiscount;
```

Comment:
The logic for applying discounts is incorrect. If the number of days is greater than or equal to the long-term threshold, the long-term discount should be applied. However, if the number of days is only greater than or equal to the short-term threshold but less than the long-term threshold, the short-term discount should be applied. The current implementation will not apply the short-term discount if the days are greater than or equal to the long-term threshold, which is correct, but the logic should ensure that only the appropriate discount is applied.